react.js

```diff
@@ -50,7 +50,7 @@ module.exports = {
     "no-unused-vars": 1,
     "react/no-unused-prop-types": 1,
     "arrow-body-style": 1,
-    "arrow-parens": 1,
+    "arrow-parens": 0,
```

Comment:
What is everyone's thoughts on this one?

Reply:
Personally, I like this rule used in conjunction with the `requireForBlockBody: true` option.